deploy/helm/trickster/templates/configmap.yaml

```diff
@@ -10,129 +10,337 @@ metadata:
 data:
   trickster.conf: |-
     [main]
-    [proxy_server]
+    {{- if .Values.configHandlerPath }}
+    config_handler_path = {{ .Values.configHandlerPath | quote }}
+    {{- end }}
+    {{- if .Values.pingHandlerPath }}
+    ping_handler_path = {{ .Values.pingHandlerPath | quote }}
+    {{- end }}
+
+    [frontend]
     listen_port = {{ .Values.service.port }}
-    {{- if .Values.listenAddress }}
-    listen_address = {{ .Values.listenAddress | quote }}
+    {{- if .Values.frontend }}
+      {{- if .Values.frontend.listenAddress }}
+    listen_address = {{ .Values.frontend.listenAddress | quote }}
+      {{- end }}
+      {{- if .Values.frontend.tlsListenPort }}
```

Comment:
For the regular `listenPort` we source that information from the service port value. Do we want to do that here as well?